src/SME.SGP.Aplicacao/Commands/EncaminhamentoAEE/ExcluirEncaminhamentoAEE/ExcluirEncaminhamentoAEECommandHandler.cs

```diff
@@ -48,8 +50,22 @@ private async Task<long> ExcluirEncaminhamentoAEE(long encaminhamentoAeeId)
                     unitOfWork.Rollback();
 
                     throw;
-                }            
+                }
             }
         }
+        private async Task ExcluirArquivos(long encaminhamentoAeeId)
+        {
+            var codigos = await repositorioEncaminhamentoAEE.ObterCodigoArquivoPorEncaminhamentoAEEId(encaminhamentoAeeId);
+            foreach (var item in codigos)
+            {
+                var entidadeArquivo = await mediator.Send(new ObterArquivoPorCodigoQuery(item.Codigo));
```

Comment:
da uma protegida nesse cara codigos
if(codigosArquivos != null && codigosArquivos.Any)
   foreach (var codigoArquivo in codigosArquivos )

Reply:
alterado